lib/init.js

```diff
@@ -0,0 +1,163 @@
+'use babel';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/extensions
+import { CompositeDisposable } from 'atom';
+
+let helpers;
+let flexPmd;
+let os;
+let fs;
+let util;
+let readFileAsync;
+let unlinkFileAsync;
+
+const REGEX = /<violation beginline="(-?\d+)" endline="(-?\d+)" begincolumn="(-?\d+)" endcolumn="(-?\d+)".* priority="(-?\d+)">(.+?)<\/violation>/g;
+
+const showErrorNotification = (text = 'Unexpected error', description) => {
+  atom.notifications.addError(`linter-flexpmd ${text}`, { description });
+};
+
+const loadDeps = () => {
+  if (!helpers) {
+    helpers = require('atom-linter');
+  }
+  if (!flexPmd) {
+    flexPmd = require('flexpmd');
+  }
+  if (!os) {
+    os = require('os');
+  }
+  if (!fs) {
+    fs = require('fs');
+  }
+  if (!util) {
+    util = require('util');
+  }
+  if (!readFileAsync) {
```

Comment:
Since `util` and `fs` are only used to implement these two dependencies let's move them into a joined `if` and `require` them in there:
```js
if (!readFileAsync || !unlinkFileAsync) {
  const fs = require('fs');
  const util = require('util');
  readFileAsync = util.promisify(fs.readFile);
  unlinkFileAsync = util.promisify(fs.unlink);
}
```